### PR TITLE
Add aliases for nom, bom and yom

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: dotfiles/.aliases [hunk 1]
```diff
@@ -47,6 +47,10 @@ alias es="ember server"
 alias et="ember test"
 alias ets="ember test --server --no-launch"
 
+alias nom="rm -rf node_modules;npm cache clean; npm i"
+alias yom="rm -rf node_modules;yarn"
+alias bom="rm -rf bower_components;bower i"
```
Comment (brzpegasus) [inline dotfiles/.aliases]: No `bower cache clean`?

Comment (ghost) [reply dotfiles/.aliases]: Didn't know that was needed, i'll add it! 😊

Comment (bcardarella) [reply dotfiles/.aliases]: can we get a combined `nombom` and `yombom` that runs both commands? Probably just:

```
alias nombom="nom && bom"
alias yombom="yom && bom"
```

Comment (ghost) [reply dotfiles/.aliases]: I opened #6 to add this.

**Pull request merged**